index.html

```diff
@@ -137,7 +137,15 @@ <h2 class="minitext">
   define("three", function() { return require("two") + require("one"); });
   define("four", function() { return require("three") + require("one"); });
 })();
-three = require("three");</code></pre>
+three = require("three");
```

Comment:
Got it. This should be done with css to make all examples on the site the same. We should handle that separately from this PR. Can you undo these blank lines?